types/chrome/index.d.ts

```diff
@@ -12840,7 +12840,7 @@ declare namespace chrome {
         export interface VpnConfigRemovalEvent extends chrome.events.Event<(id: string) => void> {}
 
         export interface VpnConfigCreationEvent
-            extends chrome.events.Event<(id: string, name: string, data: Object) => void>
```

Comment:
This is an MV2 vpn api, find it [here](https://developer.chrome.com/docs/extensions/mv2/reference/vpnProvider?hl=en#event-onConfigCreated). My admittedly limited search can't find a declaration, I'd just key-unknown this.